plugins/include/open62541/plugin/pubsub_realtime_etf.h

```diff
@@ -0,0 +1,59 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+ *
+ *    Copyright (c) 2019-2020 Kalycito Infotech Private Limited
+ */
+
+#ifndef PUBSUB_REALTIME_ETF_H_
+#define PUBSUB_REALTIME_ETF_H_
+
+#ifdef UA_ENABLE_PUBSUB_REALTIME_PUBLISH_ETF
+
+#include "ua_pubsub.h"
+#include <open62541/plugin/pubsub_udp.h>
+#ifdef UA_ENABLE_PUBSUB_ETH_UADP
+#include <open62541/plugin/pubsub_ethernet.h>
+#endif
+
+#define   PUBSUB_IP_ADDRESS                               "192.168.8.105"

```

Comment:
No. You put application specific IP addresses and cycle time in the plugin header.
As said before, this a no-go.

Please make the plugin API in a way where the configuration is provided by the application and not part of header definitions.